modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -5,9 +5,10 @@ endif::[]
 
 Use Anypoint Visualizer to view a graph of your application network.
 
-The topology of the application network is automatically generated by Anypoint Visualizer. To maximize visualization readability, consumer
- nodes (nodes that only have outbound connections), are shown at the top, and all backend systems (nodes with inbound connections only) are shown at the bottom. When the `ClientId` is detected for a consumer node, it is grouped by `ClientId` -- otherwise, it is grouped under the *External Traffic* node.
- For the `ClientId` to be detected, the target `application/API/proxy` should have the *Client Id enforcement* policy.
+Anypoint Visualizer automatically generates the topology of the application.
+To maximize visualization readability, consumer nodes (nodes that only have outbound connections) are shown at the top. Backend systems (nodes that only have inbound connections) are shown at the bottom.
+
+When the `ClientId` is detected for a consumer node, the nodes are grouped by `ClientId`; otherwise, it is grouped under the *External Traffic* node. For the `ClientId` to be detected, the target application/API/proxy should have the `ClientId` enforcement policy. For usability purposes, Client App names replace `ClientIds` in canvas. All Client Apps that call the same mule app are grouped into a single Client App group node. The number of Client Apps in the group are specified in the display name.
```

Comment:
```suggestion
When the `ClientId` is detected for a consumer node, the nodes are grouped by `ClientId`; otherwise, it is grouped under the *External Traffic* node. The target `application/API/proxy` should have the `ClientId` enforcement policy to detect the `ClientId`. For usability purposes, client app names replace `ClientIds` in the canvas. All client apps that call the same Mule app are grouped into a single client app group node. The number of client apps in the group are specified in the display name.
```